images/virtualization-artifact/pkg/controller/service/errors.go

```diff
@@ -23,3 +23,9 @@ var (
 	ErrDefaultStorageClassNotFound = errors.New("default storage class not found")
 	ErrDataVolumeNotRunning        = errors.New("pvc import is not running")
 )
+
+var (
+	ErrInvalidIpAddress      = errors.New("Invalid IP address format")
+	ErrIpAddressAlreadyExist = errors.New("IP address is already allocated")
+	ErrIpAddressOutOfRange   = errors.New("IP address is out of range")
```

Comment:
How about adding a check to the webhook to warn user?

Reply:
Maybe. It's probably worth discussing separately.

Reply:
Based on the results of the discussion, I added warn user in the webhook.